src/wp-includes/class-wpdb.php

```diff
@@ -2509,6 +2510,110 @@ public function insert( $table, $data, $format = null ) {
 		return $this->_insert_replace_helper( $table, $data, $format, 'INSERT' );
 	}
 
+	/**
+	 * Inserts multiple rows into the table in one query.
+	 *
+	 * If the insert fails, no rows will be inserted. It's not possible for some rows
+	 * to be inserted and not others.
+	 *
+	 * Examples:
+	 *
+	 *     $wpdb->insert_multiple(
+	 *         'table',
+	 *         array(
+	 *             'column1',
+	 *             'column2',
+	 *         ),
+	 *         array(
+	 *             array(
+	 *                 'column 1 value 1',
+	 *                 'column 2 value 1',
+	 *             ),
+	 *             array(
+	 *                 'column 1 value 2',
+	 *                 'column 2 value 2',
+	 *             ),
+	 *             array(
+	 *                 'column 1 value 3',
+	 *                 'column 2 value 3',
+	 *             ),
+	 *         )
+	 *     );
+	 *     $wpdb->insert_multiple(
+	 *         'table',
+	 *         array(
+	 *             'column1',
+	 *             'column2',
+	 *         ),
+	 *         array(
+	 *             array(
+	 *                 'column 1 value 1',
+	 *                 1,
+	 *             ),
+	 *             array(
+	 *                 'column 1 value 2',
+	 *                 2,
+	 *             ),
+	 *             array(
+	 *                 'column 1 value 3',
+	 *                 3,
+	 *             ),
+	 *         ),
+	 *         array(
+	 *             '%s',
+	 *             '%d',
+	 *         )
+	 *     );
+	 *
+	 * @since x.y.z
+	 *
+	 * @param string            $table   Table name.
+	 * @param list<string>      $columns Array of column names.
```

Comment:
I can't find any instances of core using `@param list`.  I think this and the next one should be arrays